.github/workflows/fluent-test-gen-all.yml

```diff
@@ -8,6 +8,14 @@ jobs:
   build:
     timeout-minutes: 180
     runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: write
+      deployments: none
+      packages: none
+      pull-requests: none
+      security-events: none
```

Comment:
fix the github bot permission
https://docs.opensource.microsoft.com/github/apps/permission-changes/